src/mlpack/tests/dcgan_test.cpp

```diff
@@ -0,0 +1,285 @@
+/**
+ * @file dcgan_network_test.cpp
+ * @author Shikhar Jaiswal
+ *
+ * Tests the DCGAN network.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/core.hpp>
+
+#include <mlpack/methods/ann/init_rules/gaussian_init.hpp>
+#include <mlpack/methods/ann/loss_functions/cross_entropy_error.hpp>
+#include <mlpack/methods/ann/loss_functions/sigmoid_cross_entropy_error.hpp>
+#include <mlpack/methods/ann/gan.hpp>
+#include <mlpack/methods/ann/ffn.hpp>
+#include <mlpack/methods/ann/layer/layer.hpp>
+#include <mlpack/methods/softmax_regression/softmax_regression.hpp>
+#include <mlpack/core/optimizers/adam/adam.hpp>
+
+#include <boost/test/unit_test.hpp>
+#include "test_tools.hpp"
+
+using namespace mlpack;
+using namespace mlpack::ann;
+using namespace mlpack::math;
+using namespace mlpack::optimization;
+using namespace mlpack::regression;
+using namespace std::placeholders;
+
+// BOOST_AUTO_TEST_SUITE(DCGANNetworkTest);
```

Comment:
The CMake/CTest configuration will try to run the `DCGANNetworkTest` when this file is added regardless, so you could either comment the file out from `CMakeLists.txt`, uncomment this line (and maybe at least one test will need to be uncommented), or uncomment the whole thing.  I think that will fix the current Travis issue.

Reply:
I commented out the file from CMakeLists.txt, it's better than letting the tests run, and potentially timeout the build.